backend/config/data_providers/api_particulier.yml

```
@@ -1032,3 +1034,36 @@ api_particulier:
           job: Directeur Business Unit adjoint 
           email: eric.rozanski@coexya.eu
           phone_number: '0487915823'
+    agedi:
+      label: >-
+        Proxima.ENF | Portail Famille
+        CNAF
```

```suggestion
        Proxima.ENF / Portail Famille,CNAF
```

Pour être consistant avec le reste